tibiapy/guild.py

```diff
@@ -8,14 +8,15 @@
 import bs4
 
 from tibiapy import abc
+from tibiapy.enums import try_enum, Vocation
 from tibiapy.errors import InvalidContent
 from tibiapy.utils import parse_tibia_date, parse_tibiadata_date
 
 COLS_INVITED_MEMBER = 2
 COLS_GUILD_MEMBER = 6
 
-founded_regex = re.compile(r'(?P<desc>.*)The guild was founded on (?P<world>\w+) on (?P<date>[^.]+)\.\nIt is (?P<status>[^.]+).',
-                           re.DOTALL)
+founded_regex = re.compile(
+    r'(?P<desc>.*)The guild was founded on (?P<world>\w+) on (?P<date>[^.]+)\.\nIt is (?P<status>[^.]+).', re.DOTALL)
```

Comment:
This is weird, PyCharm wasn't giving me any warning, I assumed it ignored that rule when it was a regex pattern.

But it's fine 👍